pkg/cmd/backtest.go

```diff
@@ -232,7 +232,10 @@ var BacktestCmd = &cobra.Command{
 							}
 
 							if prevKLine != emptyKLine {
-								if prevKLine.StartTime.Add(interval.Duration()) != k.StartTime {
+								if prevKLine.StartTime.Unix() == k.StartTime.Unix() {
+									backtestService.DeleteDuplicatedKLine(k)
```

Comment:
how this is not called internally, do you want to move this into the BacktestService as a method?

Reply:
ok